protobuf-lite/src/main/java/io/grpc/protobuf/lite/ProtoLiteUtils.java

```diff
@@ -157,12 +157,12 @@ public T parse(InputStream stream) {
         // Today the parser is always the same for a given proto, but that isn't guaranteed. Even
         // if not, using the same MethodDescriptor would ensure the parser matches and permit us
         // to enable this optimization.
-        if (protoStream.parser() == parser) {
+        if (protoStream.parser().equals(parser)) {
```

Comment:
`equals()` for `Parser` is not defined, we shouldn't use `equals()`.

Reply:
Ah okay, i change this